.github/workflows/build.yml

```diff
@@ -11,6 +11,7 @@ on:
 
 jobs:
   core:
+    runs-on: ubuntu-20.04
```

Comment:
No not really, because this just defines the GitHub Action runner. Since we are using a docker image anyhow, it really doesn't matter too much which runner we are on. I just used the `ubuntu-20.04` one because it probably is supported longer.

Reply:
Should we do `ubuntu-latest` then and never worry about it again?

Reply:
True, that one is even better!